keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -105,4 +107,10 @@ class DefaultConfiguration {
   fun authenticatedRequestFilter(): FilterRegistrationBean<AuthenticatedRequestFilter> =
     FilterRegistrationBean(AuthenticatedRequestFilter(true))
       .apply { order = HIGHEST_PRECEDENCE }
+
+  @Bean
+  fun computeResourceResolvers(resolvers: List<Resolver<*>>): List<Resolver<ComputeResourceSpec>> =
```

Comment:
Returns the list of `Resolver`s that can resolve implementations of `ComputeResourceSpec`, currently `ClusterSpec` and `TitusClusterSpec`.

Looking at this name now, I realize it's ambiguous. `Compute` was meant to be an adjective, not a verb here. Let me know if you have suggestions for a better name.